Release contact

Theia01 opened the pull request.

Commit:
Ajout de librairie

Ajout de la librairie laravelcollective pour les formulaires

Commit:
Formulaire

Création du formulaire avec bootstrap. Ainsi que de la route qui redirige vers le controller ContactUsController vers la méthode send

Dans la méthode send on vérifie si l'utilisateur à remplit correctement le formulaire avec la méthode validate qui permet la vérification (si il y a erreur, renvoie vers la page contact qui affiche les erreurs) sinon on continu, on envoie les mails (que j'explique après pour pas faire un trop gros commit). Un mail pour l'utilisateur et un pour l'administrateur

Une fois les mail envoyer un redirige vers la page contact

Commit:
Mail

Création des classes "mailable" ConfirmationContact et Contact à l'aide d'artisantion de la vue des mails

Et création des vues des mails

On fais attention lorsque l'on ajoute dans la méthode build de mettre la vue qu'il retourne avec le from

maeldebon commented on resources/views/contact.blade.php:
```diff
@@ -2,6 +2,35 @@
-    <p>contact</p>
+    <div class="container">
+
+    <h2>Contactez nous</h2>
```
J'aurais davantage vu "Contactez-nous !"

maeldebon commented on resources/views/contact.blade.php:
```diff
@@ -2,6 +2,35 @@
+    <div class="container">
+
+    <h2>Contactez nous</h2>
+    <p>Vous avez une questions, une suggestion ou une remarque. Prenez contact avec nous par ce formulaire</p>
```
"Vous avez une question, une suggestion ou une remarque ? Prenez contact avec nous via ce formulaire !"

maeldebon commented on resources/views/contact.blade.php:
```diff
@@ -2,6 +2,35 @@
+    {!! Form::open(['route'=>'sendcontact']) !!}
+        <div class="form-group {{ $errors->has('name') ? 'has-error' : '' }}">
+            {!! Form::label('Prénom:') !!}
+            {!! Form::text('name', old('name'), ['class'=>'form-control', 'placeholder'=>'Entrer un nom']) !!}
```
"Entrez votre prénom"

maeldebon commented on resources/views/contact.blade.php:
```diff
@@ -2,6 +2,35 @@
+            <span class="text-danger">{{ $errors->first('message') }}</span>
+        </div>
+        <div class="form-group">
+            <button class="btn btn-success">Contact US!</button>
```
"Envoyer ce message"

maeldebon commented on resources/views/contact.blade.php:
```diff
@@ -2,6 +2,35 @@
+        </div>
+        <div class="form-group {{ $errors->has('message') ? 'has-error' : '' }}">
+            {!! Form::label('Message:') !!}
+            {!! Form::textarea('message', old('message'), ['class'=>'form-control', 'placeholder'=>'Entrer un message']) !!}
```
"Entrez votre message"

maeldebon commented on resources/views/contact.blade.php:
```diff
@@ -2,6 +2,35 @@
+
+
 </div>
 @endsection
```
Il manque une ligne vide à la fin du fichier 🙂

maeldebon commented on resources/views/contact.blade.php:
```diff
@@ -2,6 +2,35 @@
+        </div>
+        <div class="form-group {{ $errors->has('email') ? 'has-error' : '' }}">
+            {!! Form::label('Email:') !!}
+            {!! Form::text('email', old('email'), ['class'=>'form-control', 'placeholder'=>'Entrer un email']) !!}
```
J'aurais plus vu "Entrez votre adresse mail" au niveau du placeholder 🙂

maeldebon reviewed (changes requested):
Très bon taff ! Quelques petites modifs et ce sera parfait pour moi ! 🙂

Commit:
Séparer vues emails

Après quelques rectification d’écriture, j'ai séparer les vues pour en faire une général qui sert de template pour les deux autres. Ce qui permet d'éviter la répétition et pour chaque nouveau mail, il seront quasi-identique

maeldebon commented on resources/views/emails/confirmation.blade.php:
```diff
@@ -0,0 +1,9 @@
+    Nous répondrons à votre demande dans les plus brefs délais !
+    <br><br>
+    Camille de Moricet Massage.</p>
+@endsection
```
Il manque un linebreak :)

Commit:
Un espace

Pour la propreté du code

maeldebon reviewed (approved):
C'est bon pour moi !
